app/(app)/tools/sleep/sleep-habits/awake-activities.tsx

```diff
@@ -0,0 +1,80 @@
+import { Icon } from '@/components/Icon';
+import React, { useMemo, useState } from 'react';
+import { Screen } from '@/components/Screen';
+import useTranslationKeys from '@/hooks/useTranslationKeys';
+import { useTranslation } from 'react-i18next';
+import { router } from 'expo-router';
+import sleepActivitiesRepository, { SleepActivityType } from '@/db/repositories/sleep-activities.repository';
+import StarList from '@/components/StarList';
+import { useSleepActivitiesByType } from '@/services/sleep-activities.service';
+
+export default function AwakeActivities() {
+  const { t } = useTranslation('tools');
+  const { toolsTranslationKeys } = useTranslationKeys();
+  const { data: sleepActivity, refetch } = useSleepActivitiesByType(SleepActivityType.AWAKE);
+  const [favoriteItems, setFavoriteItems] = useState<string[]>([]);
+
+  const activities = useMemo(() => {
```

Comment:
Ma asteptam sa vad aici maparea cu favorite true/false si mi-a luat ceva sa ma uit in StarList.